docs/guides/help-for-the-non-technical.md

```diff
@@ -0,0 +1,196 @@
+# Help for our less-technical users
```

Comment:
It would be good to get feedback from less-technical users how they react to this title. An alternative could be "Help for non-developers" (or similar)?

Reply:
I actually like 'Help for non-developers' so I'm going to steal it!